test/unit/architecture_test.rb

```diff
@@ -25,28 +25,53 @@ class ArchitectureTest < ActiveSupport::TestCase
     assert_not architecture.destroy
   end
 
-  test "should return intel precision for i386" do
+  test "should return EFI filename for i386" do
     architecture = Architecture.new :name => "i386"
-    assert_equal "ia32", architecture.intel_precision
+    assert_equal "ia32", architecture.bootfilename_efi
   end
 
-  test "should return intel precision for i686" do
+  test "should return EFI filename for i686" do
     architecture = Architecture.new :name => "i686"
-    assert_equal "ia32", architecture.intel_precision
+    assert_equal "ia32", architecture.bootfilename_efi
   end
 
-  test "should return intel precision for x86-64" do
+  test "should return EFI filename for x86-64" do
     architecture = Architecture.new :name => "x86-64"
-    assert_equal "x64", architecture.intel_precision
+    assert_equal "x64", architecture.bootfilename_efi
   end
 
-  test "should return intel precision for x86_64" do
+  test "should return EFI filename for x86_64" do
     architecture = Architecture.new :name => "x86_64"
-    assert_equal "x64", architecture.intel_precision
+    assert_equal "x64", architecture.bootfilename_efi
```

Comment:
took me a split second to not raise an eyebrow and think this and the previous test were the same :)